torch/fx/experimental/migrate_gradual_types/constraint_generator.py

```diff
@@ -33,31 +35,244 @@ def generate_flatten_constraints(start_dim, end_dim, input, flattened, n, counte
     return Conj([c1, c2, *nat_constraints]), counter
 
 
-@register_inference_rule(torch.add)
-@register_inference_rule(operator.add)
-def add_inference_rule(n: Node, symbols, constraints, counter):
+# TODO
+@register_inference_rule("long")
+def long_inference_rule(n: Node, symbols, constraints, counter):
+    """
+    """
+    return [], counter
+
+# TODO
```

Comment:
Maybe `throw NotImplementedError` in the body of these TODO functions?

Reply:
done